chunkey/encode_pipeline.py

```diff
@@ -282,7 +272,7 @@ def _determine_bandwidth(self, profile_name):
         """
         max_bandwidth = 0.0
 
-        for file in os.listdir(self.video_root):
+        for file in os.listdir(self.video_root):  # pylint: disable=redefined-builtin
```

Comment:
We could clean this up a bit...just use a different variable name besides `file` here and inside the loop, and we can remove this pylint statement.